spec/src/main/asciidoc/changelog.adoc

```diff
@@ -25,14 +25,28 @@
 [[release_notes_5_0]]
 == Changes in 5.0
 
+=== Incompatible Changes
+** This release aligns with Jakarta EE 10, so it won’t work with earlier versions of Jakarta or Java EE
+
 === Breaking changes
+* Removed SimpleTimer class and SimplyTimed annotation
+* Removed ConcurrentGauge class and ConcurrentGauge annotation
+* Removed Meter class and Metered annotation
+* Removed Metered interface
 
-=== API/SPI Changes
 * Updated Timer class
 ** Removed `getFifteenMinuteRate()` method
 ** Removed `getFiveMinuteRate()` method
-** Removed `getMeanMinuteRate()` method
+** Removed `getMeanRate()` method
+** Removed `getOneMinuteRate()` method
+** Removed `getStdDev()` method
+
+* Updated Histogram class
+** Removed `getFifteenMinuteRate()` method
```

Comment:
Nothing was Changed in Histogram. There aren't any rate methods in Histogram.

Reply:
oh, good point.